src/Elmish.WPF/BindingVmHelpers.fs

```diff
@@ -236,15 +236,17 @@ type FirstValidationErrors() =
 
 type FuncsFromSubModelSeqKeyed() =
 
-  member _.Base(binding: BaseVmBinding<'model, 'msg>) =
+  member _.Base(binding: BaseVmBinding<'model, 'msg, 'a>): SelectedItemBinding<obj, obj, 'a, obj> option =
     match binding with
-    | SubModelSeqKeyed b -> Some { FromId = b.FromId
-                                   VmToId = b.SubModelSeqKeyedData.VmToId }
+    | SubModelSeqKeyed b ->
+      { VmToId = box >> b.SubModelSeqKeyedData.VmToId
+        FromId = b.FromId >> Option.map unbox }
+      |> Some
```

Comment:
Please extract the refactors in there to a previous commit

Reply:
Reverted this refactor

Reply:
#498